presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -399,14 +400,18 @@ public void testSplitsGenerationWithAbortedTransactions()
                         "transactional_properties", "insert_only"));
 
         List<String> filePaths = ImmutableList.of(
+                tablePath + "/delta_0000001_0000001_0000/_orc_acid_version",
                 tablePath + "/delta_0000001_0000001_0000/bucket_00000",
+                tablePath + "/delta_0000002_0000002_0000/_orc_acid_version",
                 tablePath + "/delta_0000002_0000002_0000/bucket_00000",
+                tablePath + "/delta_0000003_0000003_0000/_orc_acid_version",
                 tablePath + "/delta_0000003_0000003_0000/bucket_00000");
 
         for (String path : filePaths) {
             File file = new File(path);
-            assertTrue(file.getParentFile().mkdirs(), "Failed creating directory " + file.getParentFile());
+            assertTrue(file.getParentFile().exists() || file.getParentFile().mkdirs(), "Failed creating directory " + file.getParentFile());
             assertTrue(file.createNewFile(), "Failed to create file");
+            writeDeltaFileData(file);
```

Comment:
`createNewFile` on the line above is now redundant

Reply:
fixed.